### PR TITLE
perf: reduce fuel persistency update rate and enable clang optimizations

### PR DESCRIPTION
**Pull request by Saschl**
Fixes #[issue_no]

## Summary of Changes

I noticed that the file for fuel persistency was written every update cycle while being on ground which showed quite a high runtime in the WASM debug window. I've added a timer to only update it every second which should be sufficient IMO.

Furthermore I've enabled clang optimization which also showed a slight improvement in runtime. Note that I used O3 which might be aggressive but I've done a few flights over the last week and didn't notice any erratic behaviour.

### Before

fadec_before
fbw_before

### After
fadec_after
fbw_after

## Additional context

Discord username (if different from GitHub):

## Testing instructions

1. Spawn C&D and verify the fuel state is saved correctly every second (check the Modified timestamp of the file in the "work/AircraftStates" for the selected livery.
2. Take off and verify the fuel state is not updated (as it was before this change)
3. Observe system behaviour (FBW, AP and Engines) for any possible strange behaviour (very high or low numbers, unusual behaviour)

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/fadec/src/EngineControl.h [hunk 1]
```diff
@@ -20,11 +20,11 @@
 
 /* Values in gallons */
 struct Configuration {
-    double fuelCenter = 0;
```
Comment (Saschl) [inline src/fadec/src/EngineControl.h:23]: Looks like VSCode applied formatting in some places.. Maybe makes sense at some places but I can revert that if needed.

**Comment by Saschl** [comment]
CC @Taz5150

**Comment by Taz5150** [comment]
> CC @Taz5150

Sorry, did not see this message. Yes please, feel free to make pertinent changes.

**Comment by Saschl** [comment]
Done @2hwk

**Review by 2hwk — APPROVED** [review]

**Comment by 2hwk** [comment]
Tests passed:
- [x] Spawn C&D and verify the fuel state is saved correctly every second (check the Modified timestamp of the file in the "work/AircraftStates" for the selected livery.
- [x] Take off and verify the fuel state is not updated (as it was before this change)
- [x] Observe system behaviour (FBW, AP and Engines) for any possible strange behaviour (very high or low numbers, unusual behaviour)

**Pull request merged**